libs/wingc/src/parser.rs

```diff
@@ -262,7 +280,10 @@ fn parse_wing_file(
 
 	// Parse the source text into an AST
 	let parser = Parser::new(&source_text.as_bytes(), source_file.to_owned());
-	let (scope, dependent_wing_paths) = parser.parse(&tree_sitter_root);
+	let (scope, dependent_wing_paths, found_library_roots) = parser.parse(&tree_sitter_root);
+	for (name, path) in found_library_roots {
+		library_roots.insert(name, path);
+	}
```

Comment:
My end goal here was when we're done with parsing, we should have a list of all of the libraries we import (plus the current project or library) and their root paths, so we can use that info in the type checker for calculating FQNs.

Now where do we actually create or generate that data? It's not that clear what the easiest way to add this was -- I ended up just passing it around in the parser functions we have here, but I could imagine it might be better if we had some more flexible API that let us just query for this information or just hand the parser a callback that should be called every time a new library is encountered, etc. It might be worthwhile to see how other compilers handle this.